src/routes/index.tsx

```diff
@@ -38,30 +41,42 @@ function Index() {
   return (
     <div>
       <h1 className="text-2xl">
-        Welcome to <strong>Bookracy</strong> 📚
+        Welcome to <strong className="text-primary">Bookracy</strong> 📚
       </h1>
+      <br />
       <div className="flex flex-col gap-6">
         <div className="flex flex-col gap-2">
           <p>
             Bookracy is a free and open-source web app that allows you to read and download your favorite books, comics, and manga.
             <br />
-            To get started, either search below or navigate the site using the sidebar.
+            To get started, search below or navigate the site using the sidebar or click to <NavLink to="/about">Learn More →</NavLink>
           </p>
-          <NavLink to="/about">About Us</NavLink>
         </div>
 
-        <Input
-          className="w-3/4 transition-shadow duration-300 focus-visible:shadow-md md:w-4/6"
-          placeholder="Search for books, comics, or manga..."
-          value={q}
-          onChange={(e) =>
-            navigate({
-              search: {
-                q: e.target.value,
-              },
-            })
-          }
-        />
+        {/* Sticky Search Input */}
+        <div
+          className={cn("sticky left-[40px] top-[8px] z-50 w-full", {
+            "left-[200px]": sidebar.isOpen,
+          })}
+        >
+          <div className="relative w-full max-w-full">
+            {" "}
+            {/* Adjust max width as needed */}
+            <SearchIcon className="absolute left-3 top-1/2 h-4 w-4 -translate-y-1/2 text-muted-foreground" />
+            <Input
+              className="flex h-10 rounded-md border border-input bg-background px-10 py-2 text-sm transition-all duration-300 focus-visible:shadow-md focus-visible:ring-2 focus-visible:ring-primary/50"
```

Comment:
Are we sure all these styles are needed and the base Input styling isn't sufficient?